components/expenses/PayoutBankInformationForm.js

```diff
@@ -362,6 +362,15 @@ const PayoutBankInformationForm = ({ isNew, getFieldName, host, fixedCurrency, i
   // Display spinner if loading
   if (loading) {
     return <StyledSpinner />;
+  } else if (host.slug === TW_API_COLLECTIVE_SLUG && !data?.host?.transferwise) {
+    return (
+      <MessageBox fontSize="12px" type="warning">
+        <FormattedMessage
+          id="PayoutBankInformationForm.Warning.WiseNotConfigured"
+          defaultMessage="Wise is not configured on the platform."
+        />
```

Comment:
I don't think you need to internationalize since this happens only to poorly configurated environments.

Reply:
Makes sense. I've removed the internationalization for this. Thanks. 👍🏽